### PR TITLE
Document how to do update/maintenance

### PR DESCRIPTION
**Pull request by corvus-ch**

### CODE REVIEW

File: docs/modules/ROOT/pages/how-tos/update_maintenance.adoc [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+. Get list of available updates:
++
+[source,console]
+----
+kubectl --as cluster-admin get clusterversion version -o json | jq '.status.availableUpdates[] | {image: .image, version: .version}'
+----
+       
+. Update the configuration hierarchy
++
+Set the follwoing parameters to the values retrieved in the previous step:
```
Comment (ccremer) [inline docs/modules/ROOT/pages/how-tos/update_maintenance.adoc:12]:
```suggestion
Set the following parameters to the values retrieved in the previous step:
```

**Review by ccremer — CHANGES_REQUESTED** [review]

**Review by ccremer — APPROVED** [review]
I cannot judge the procedure itself though

**Review by tobru — APPROVED** [review]

File: docs/modules/ROOT/pages/how-tos/update_maintenance.adoc [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+* `parameters.openshift4_version.spec.desiredUpdate.image`
+* `parameters.openshift4_version.spec.desiredUpdate.version`
+
+. Compile the cluster catalog
+
+. Enjoy the show
++
+Let the OpenShift operators do their job.
+
+See also https://docs.openshift.com/container-platform/latest/updating/updating-cluster-between-minor.html[Updating a cluster between minor versions] of the OpenShift 4 documentation.
```
Comment (ccremer) [inline docs/modules/ROOT/pages/how-tos/update_maintenance.adoc:23]: How do I enjoy the show?
The article linked does not really tell how I can follow the update. yes it mentions the machine config operator, but what do I need to look out for? How can I verify whether the upgrade worked and is finished or failed? Or are these things of the past?

Comment (corvus-ch) [reply docs/modules/ROOT/pages/how-tos/update_maintenance.adoc:23]: I do lack the experience to write down something that works. My intention was to improve later.

Comment (ccremer) [reply docs/modules/ROOT/pages/how-tos/update_maintenance.adoc:23]: Ok. Maybe add the line `kubectl --as cluster-admin get clusterversion version --watch` for the moment, as it displays at least whether it's progressing, faulting or done. And a nice sentence

Comment (corvus-ch) [reply docs/modules/ROOT/pages/how-tos/update_maintenance.adoc]: Done.

File: docs/modules/ROOT/pages/how-tos/update_maintenance.adoc [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
++
+Let the OpenShift operators do their job.
++
+[source,console]
+----
+kubectl --as cluster-admin get clusterversion version --watch
+----
+
+So far, that process just worked.
+In consequence we do lack the experience on how to observe and debugg.
```
Comment (ccremer) [inline docs/modules/ROOT/pages/how-tos/update_maintenance.adoc:29]:
```suggestion
In consequence we do lack the experience on how to observe and debug.
```

**Review by ccremer — CHANGES_REQUESTED** [review]

**Review by ccremer — APPROVED** [review]

**Pull request merged**